app/src/main/java/com/hedvig/app/feature/onboarding/GetBundlesUseCase.kt

```diff
@@ -6,23 +6,32 @@ import com.hedvig.android.owldroid.type.EmbarkStoryType
 import com.hedvig.app.util.LocaleManager
 import com.hedvig.app.util.apollo.QueryResult
 import com.hedvig.app.util.apollo.safeQuery
+import com.hedvig.app.util.featureflags.Feature
+import com.hedvig.app.util.featureflags.FeatureManager
 
 class GetBundlesUseCase(
     private val apolloClient: ApolloClient,
-    private val localeManager: LocaleManager
+    private val localeManager: LocaleManager,
+    featureManager: FeatureManager,
 ) {
+    private val storyType = if (featureManager.isFeatureEnabled(Feature.QUOTE_CART)) {
+        EmbarkStoryType.APP_ONBOARDING_QUOTE_CART
+    } else {
+        EmbarkStoryType.APP_ONBOARDING
+    }
+
```

Comment:
Yes you are right. Fixing that

Reply:
Do you have some other way to solve this nicer for you instead of just always having that change not committed? It'll be quite annoying to always have to not commit 1 file

Reply:
Good question, dont have a solution to that 🤔

Reply:
We should have some kind of engineering mode in the test app where you can manage feature flags